src/Index.php

```diff
@@ -96,6 +96,10 @@ public function search(array $searchParams, ResultTransformer $resultTransformer
 
         if ($resultTransformer) {
             $result = $resultTransformer->formatResult($result);
+            if ($result->getLimit() === null) {
```

Comment:
Does not the limit should be passed to the transformer ?

Reply:
I don't understand why the limit is lost during the transformation ?

Reply:
As discussed internally, bu for the record : Result is a Immutable object, which

Reply:
👍